app/config.py

```diff
@@ -16,11 +17,18 @@ class Settings(BaseSettings):
     db_port: str
     db_password: str
     db_user: str
-    secret_key: str
+
+    refresh_token_name: str = "refresh_token"
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (complexity):** Consider using inheritance to manage the complexity of the `Settings` class.

The introduction of new settings related to token management significantly increases the complexity of the `Settings` class. This is due to the addition of multiple new variables (`refresh_token_name`, `access_token_name`, `verify_token_secret_key`, `access_token_secret_key`, `refresh_token_secret_key`, `token_audience`), the use of a `List` type for `token_audience`, and the overall increase in the number of environment variables that need to be managed. While these additions might be necessary for the application's functionality, it's important to consider strategies to manage this complexity effectively.

One approach to consider is the use of inheritance to group related settings, which can help in organizing the code better and making it easier to understand. For example, you could create base classes for token-related settings and then extend these for specific token types (access, refresh). This not only reduces repetition but also logically groups settings, making the class more maintainable and easier to navigate.

Here's a simplified example of how you might structure this using inheritance:

```python
from typing import List
from pydantic_settings import BaseSettings

class BaseTokenSettings(BaseSettings):
    secret_key: str
    expire_minutes: int

class AccessTokenSettings(BaseTokenSettings):
    token_name: str = "access_token"
    expire_minutes: int = 30

class RefreshTokenSettings(BaseTokenSettings):
    token_name: str = "refresh_token"
    expire_minutes: int = 1440

class Settings(BaseSettings):
    app_name: str = "pecuny"
    max_allowed_accounts: int = 5
    environment: str = "dev"
    # Other settings...
    access_token: AccessTokenSettings = AccessTokenSettings()
    refresh_token: RefreshTokenSettings = RefreshTokenSettings()
    token_audience: List[str] = ["fastapi-users:auth"]
    # Other settings...
```

This approach not only makes the `Settings` class cleaner but also encapsulates token-related settings in a more coherent manner. Consider whether a similar strategy could be applied to manage the complexity introduced by the new settings.